tock/tock/views.py

```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render_to_response
+
+
+def csrf_failure(request, reason=""):
+    return render_to_response('403.html')
```

Comment:
I wonder if it'd be useful to include an extra bit of info for devs in the final view?  For instance, if I'm working on Tock and I get Django's default CSRF failure view, I'll understand what I need to do to fix things, but if I get a 403 page, I'll start thinking about permissions, which is actually the wrong road to go down when debugging this problem.

What if our error pages had a "technical details" section that could include technical jargon useful for debugging?  (This could be something that only displays when `DEBUG` is active, but it might also be useful to have in production, in case end-users happen to run into it...)

Reply:
This may just be my general dislike of render_to_response, but I think we should use render, if only because I think at some point we may get tripped up wondering why expected context ain't in the template.